keyboards/zos/65s/info.json

```
@@ -0,0 +1,100 @@
+{
+    "manufacturer": "ZOS",
+    "keyboard_name": "ZOS 65s",
+    "maintainer": "Zenn",
+    "bootloader": "atmel-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": false,
```

This is preferred to be left enabled, if possible.
```suggestion
        "mousekey": true,
```

---

Thanks done.